apps/src/block_utils.js

```diff
@@ -996,6 +1000,32 @@ exports.createJsWrapperBlockCreator = function(
           this.setPreviousStatement(true);
         }
 
+        if (
+          miniToolboxBlocks &&
+          experiments.isEnabled(experiments.MINI_TOOLBOX)
+        ) {
+          var toggle = new Blockly.FieldIcon('+');
+          var miniToolboxXml = '<xml>';
+          miniToolboxBlocks.forEach(block => {
+            miniToolboxXml += `\n <block type="${block}"></block>`;
+          });
+          miniToolboxXml += '\n</xml>';
+          this.tray = false;
+          Blockly.bindEvent_(toggle.fieldGroup_, 'mousedown', this, () => {
```

Comment:
NIT: Mind leaving a quick comment explaining what's going on here? It makes sense next to the visual, but is less obvious when reading the code directly.